### PR TITLE
adding the logos of Afnic foundation and defensive lab agency

### PR DESCRIPTION
**Pull request by MeTaLPoU**

### CODE REVIEW

**Review by codeurimpulsif — CHANGES_REQUESTED** [review]
DLA logo is ok but the AFNIC one is small, maybe we need to resize the image to reduce blank space in it.

File: content/page/supporters.en.md [hunk 1]
```diff
@@ -31,6 +39,14 @@ draft: false
             </div>
         </a>
     </div>
+    <div class="col-lg-4 col-sm-6 text-center">
+        <a href="https://www.fondation-afnic.fr">
+            <img src="/media/partners/fondation-Afnic.png" width="120px" height="120px"class="mt-3 ml-auto mr-auto"/>
+            <div class="card-body">
+                <h6 class="card-title">Fondation Afnic pour la solidarité numérique</h6>
```
Comment (pnu-s) [inline content/page/supporters.en.md:46]:
```suggestion
                <h6 class="card-title">AFNIC foundation</h6>
```

File: content/page/supporters.fr.md [hunk 2]
```diff
@@ -31,6 +39,14 @@ draft: false
             </div>
         </a>
     </div>
+    <div class="col-lg-4 col-sm-6 text-center">
+        <a href="https://www.fondation-afnic.fr">
+            <img src="/media/partners/fondation-Afnic.png" width="120px" height="120px"class="mt-3 ml-auto mr-auto"/>
+            <div class="card-body">
+                <h6 class="card-title">Fondation Afnic pour la solidarité numérique</h6>
```
Comment (pnu-s) [inline content/page/supporters.fr.md:46]:
```suggestion
                <h6 class="card-title">Fondation AFNIC</h6>
```

**Review by pnu-s — CHANGES_REQUESTED** [review]

**Review by codeurimpulsif — APPROVED** [review]
I think we can merge this :smiley:

**Review by pnu-s — APPROVED** [review]

**Pull request merged**